cns/NetworkContainerContract.go

```diff
@@ -63,12 +68,14 @@ type CreateNetworkContainerRequest struct {
 	Routes                     []Route
 	AllowHostToNCCommunication bool
 	AllowNCToHostCommunication bool
+	EndpointPolicies           []NetworkContainerRequestPolicies
```

Comment:
Should this be named as NetworkContainerPolicies?

Reply:
I think we can leave it as is. Its to me easier to tell between the policies in the request and the static ones we have throughout the code. But will change if you feel otherwise.

Reply:
no, that's fine. thanks